src/components/chip-set/chip-set.tsx

```diff
@@ -21,6 +21,7 @@ import translate from '../../global/translations';
 import { getHref, getTarget } from '../../util/link-helper';
 
 const SELECTED_CHIP_CLASS = 'mdc-chip--selected';
+const INPUT_FIELD_TABINDEX = 1;
```

Comment:
@jensgustafsson @Kiarokh I came across this when doing other stuff in this file, and thought I'd take a moment to spread some knowledge 😊

From [MDN's docs for tabindex](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex):

<img width="761" alt="image" src="https://user-images.githubusercontent.com/154725/153171309-b7ec227e-1cb2-4695-82c8-a0bb2660a474.png" alt="A positive value means the element should be focusable in sequential keyboard navigation, with its order defined by the value of the number. That is, tabindex=4 is focused before tabindex=5 and tabindex=0, but after tabindex=3. If multiple elements share the same positive tabindex value, their order relative to each other follows their position in the document source. The maximum value for tabindex is 32767. If not specified, it takes the default value 0. Warning: Avoid using tabindex values greater than 0. Doing so makes it difficult for people who rely on assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.">

However, I tested this in the web client, and it didn't seem to have that effect, so it's possible that the above doesn't apply between different shadow roots… 🤔

Anyway, it's probably for the best not to use `tabindex=1`… 🙂